core/templates/pages/exploration-editor-page/services/version-history.service.spec.ts

```diff
@@ -681,7 +681,7 @@ describe('Version history service', () => {
     () => {
       expect(
         versionHistoryService
-          .getCurrentPositionInStateVersionHistoryList()).toBeNull();
+          .getCurrentPositionInStateVersionHistoryList()).toEqual(0);
```

Comment:
Just to check, can you explain why this changed? (Doing so in these github review comments is fine)

Reply:
It is because earlier, the initial values of currentPositionInStateVersionHistoryList and currentPositionInMetadataVersionHistoryList were null and their types were (number | null). After strongly typing those variables, I set the initial values of these variables to 0.